release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -2274,6 +2287,8 @@ For more information, see xref:../networking/ovn_kubernetes_network_provider/con
 
 * Adding or removing nodes from the cluster can cause ownership contention over the node status. This can cause new nodes to take an extended period of time to appear. As a workaround, you can restart the `kube-apiserver-operator` pod in the `openshift-kube-apiserver-operator` namespace to expedite the process. (link:https://issues.redhat.com/browse/OCPBUGS-50587[*OCPBUGS-50587*])
 
+* For dual-stack networking clusters that run on {rh-openstack}, when a Virtual IP (VIP) that is attached to a Floating IP (FIP) moves between master nodes, the association between VIP and FIP may stop working if the new master is on a different compute node. This issue occurs because OVN assumes that both IPv4 and IPv6 addresses on a shared Neutron port belong to the same node. (link:https://issues.redhat.com//browse/OCPBUGS-50599[*OCPBUGS-50599*])
```

Comment:
```suggestion
* For dual-stack networking clusters that run on {rh-openstack}, when a Virtual IP (VIP) that is attached to a Floating IP (FIP) moves between master nodes, the association between VIP and FIP might stop working if the new master is on a different compute node. This issue occurs because OVN assumes that both IPv4 and IPv6 addresses on a shared Neutron port belong to the same node. (link:https://issues.redhat.com//browse/OCPBUGS-50599[*OCPBUGS-50599*])
```